packages/webapp/src/containers/taskSlice.js

```diff
@@ -237,8 +237,10 @@ export const taskEntitiesSelector = createSelector(
           taskEntities[task_id].managementPlans?.map(getManagementPlanByPlantingManagementPlan) ||
           [];
         taskEntities[task_id].locations =
-          taskEntities[task_id].locations?.map((location_id) => locationEntities[location_id]) ||
-          [];
+          // TODO: LF-4709 Revert logic once sensors are no longer included in the backend response
+          taskEntities[task_id].locations?.flatMap(
+            (location_id) => locationEntities[location_id] || [],
+          ) || [];
```

Comment:
This removes tasks that contain a sensor location and require a location? (ie cleaning an pest control, but not custom?) 

This has me curious for backend part about removing a taskLocation from a task requiring a location.... but we can discuss on backend ticket.